integration/simulation/network/network.go

```diff
@@ -44,8 +47,15 @@ func (n *RPCHandles) RndObscuroClient() rpcclientlib.Client {
 	return n.ObscuroClients[rand.Intn(len(n.ObscuroClients))] //nolint:gosec
 }
 
-// ObscuroWalletClient fetches client for given wallet if it exists
-func (n *RPCHandles) ObscuroWalletClient(wallet wallet.Wallet) rpcclientlib.Client {
+// ObscuroWalletRndClient fetches a client for a given wallet connected to a random node
```

Comment:
@BedrockSquirrel Maybe clearer as:

`// ObscuroWalletRndClient fetches an RPC client connected to a random L2 node for a given wallet`

? Sorry!